application/dash_application/views/single_lane_cfmedip.py

```diff
@@ -74,8 +78,17 @@
 initial["second_sort"] = CFMEDIP_COL.RelativeCpGFrequencyEnrichment
 initial["institutes"] = []
 # Set initial values for graph cutoff lines
-cutoff_pf_reads_label = sidebar_utils.total_reads_cutoff_label
-initial["cutoff_pf_reads"] = 0.01
+# TODO: is this just clusters-per-sample worded differently?
```

Comment:
@slazicoicr

Reply:
Yes. Currently, the cluster column (from merging with FastQC) has not been done for this view.